src/commands/workflow/run.ts

```diff
@@ -44,4 +50,40 @@ export default class WorkflowRun extends BaseCommand {
       `‣ Successfully ran \`${args.workflowKey}\` workflow in \`${flags.environment}\` environment. Run id: ${resp.data.workflow_run_id}.`,
     );
   }
+
+  async runLocal(): Promise<void> {
+    const target = await Workflow.ensureValidCommandTarget(
+      this.props,
+      this.runContext,
+    );
+
+    const { type: targetType, context: targetCtx } = target;
+
+    if (!targetCtx.exists) {
+      const subject =
+        targetType === "workflowDir"
+          ? "a workflow directory at"
+          : "workflow directories in";
```

Comment:
Related to the comment above, we can just write "a workflow directory at" directly into the error message. The other one is for when the command target is for "workflowsIndexDir", which is not supported for this command.